rclcpp_components/src/component_container_mt.cpp

```diff
@@ -21,8 +21,52 @@
 int main(int argc, char * argv[])
 {
   /// Component container with a multi-threaded executor.
-  rclcpp::init(argc, argv);
-  auto exec = std::make_shared<rclcpp::executors::MultiThreadedExecutor>();
+  auto vargv = rclcpp::init_and_remove_ros_arguments(argc, argv);
+  size_t number_of_threads{0};
+  rclcpp::Logger logger{rclcpp::get_logger("component_container_mt")};
+
+  if (vargv.size() > 1) {
+    for (auto itr = vargv.begin() + 1; itr != vargv.end(); ++itr) {
+      if (*itr == "--help" || *itr == "-h") {
+        RCLCPP_INFO_STREAM(
+          logger,
+          "Usage: component_container_mt --thread-num <number of thread>");
+        return 0;
+      } else if (*itr == "--thread-num" || *itr == "-t") {
+        if (itr == vargv.end() - 1) {
+          RCLCPP_ERROR_STREAM(
+            logger, "Empty thread number" << std::endl <<
+              "Usage: component_container_mt --thread-num <number of thread>");
+          return 0;
+        }
+        try {
+          const auto arg_number_of_threads{std::stoi(*(itr + 1))};
```

Comment:
`std::stoull()` seems more appropriate

```suggestion
          const auto arg_number_of_threads{std::stoull(*(itr + 1))};
```

Reply:
@ivanpauno
`stoull` cannot handle a minus value which is converted to a huge number. How do you think?

Reply:
that's interesting ....
I thought that would throw `std::invalid_argument{}`, but the cpp standard library always manages to surprise me.

Reply:
resolving this comment, only adding unit tests is pending then